### PR TITLE
(MODULES-8108) - Support added for Debian 9

### PR DESCRIPTION
**Pull request by david22swan**
Updated default params added.
Exclusions added for certain features.

### CODE REVIEW

**Comment by HelenCampbell** [comment]
@david22swan Rubocop pls!

**Comment by david22swan** [comment]
@HelenCampbell Rubocop'ed!

**Comment by david22swan** [comment]
screen shot 2018-11-21 at 3 44 24 pm

File: manifests/mod/dav_svn.pp [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   ::apache::mod { 'dav_svn': }
 
-  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and $::operatingsystemmajrelease != '10.04' and $::operatingsystemrelease != '10.04' and $::operatingsystemmajrelease != '16.04') {
+  if $::osfamily == 'Debian' and ($::operatingsystemmajrelease != '6' and $::operatingsystemmajrelease != '10.04' and $::operatingsystemrelease != '10.04' and $::operatingsystemmajrelease != '16.04' and $::operatingsystemmajrelease != '9') {
```
Comment (ekohl) [inline manifests/mod/dav_svn.pp]: How about: 
```puppet
if $::osfamily == 'Debian' and $::operatingsystemrelease != '10.04' and ! ($::operatingsystemmajrelease in ['6', '10.04', '16.04', '9']) {
```

And maybe just drop 10.04 here :)

Comment (david22swan) [reply manifests/mod/dav_svn.pp]: @ekohl Updated :). Considered removing Deb 6 as well but as it's not as out of date decided to leave for now.

Comment (ekohl) [reply manifests/mod/dav_svn.pp]: Slightly easier to read now :)

**Comment by pmcmaw** [comment]
@ekohl are you happy enough with this PR? 
I don't want to cut your review short, but when you are happy I will merge. I have had a look and everything looks fine to me :-)

**Comment by david22swan** [comment]
@ekohl Got an Ubuntu PR coming soon after this one.

**Pull request merged**